src/overlays/actors/ovl_En_Lift_Nuts/z_en_lift_nuts.c

```diff
@@ -28,111 +62,930 @@ const ActorInit En_Lift_Nuts_InitVars = {
+                switch (CURRENT_DAY) {
+                    case 1:
+                        if (gSaveContext.save.time > CLOCK_TIME(23, 30) || gSaveContext.save.time <= CLOCK_TIME(6, 0)) {
+                            Message_StartTextbox(play, 0x27F7, &this->actor);
+                            this->textId = 0x27F7;
+                        } else if (gSaveContext.save.weekEventReg[14] & 0x10) {
+                            Message_StartTextbox(play, 0x27D9, &this->actor);
+                            this->textId = 0x27D9;
+                        } else {
+                            Message_StartTextbox(play, 0x27DA, &this->actor);
+                            this->textId = 0x27DA;
+                        }
+                        break;
+                    case 2:
+                        if (gSaveContext.save.time > CLOCK_TIME(23, 30) || gSaveContext.save.time <= CLOCK_TIME(6, 0)) {
+                            Message_StartTextbox(play, 0x27F7, &this->actor);
+                            this->textId = 0x27F7;
+                        } else {
+                            if (gSaveContext.save.weekEventReg[14] & 0x20) {
+                                Message_StartTextbox(play, 0x27DB, &this->actor);
+                                this->textId = 0x27DB;
+                                break;
+                            }
+
+                            if (gSaveContext.save.weekEventReg[14] & 0x10) {
+                                Message_StartTextbox(play, 0x27DC, &this->actor);
+                                this->textId = 0x27DC;
+                            } else {
+                                Message_StartTextbox(play, 0x27DD, &this->actor);
+                                this->textId = 0x27DD;
+                            }
+                        }
+                        break;
+                    case 3:
+                        if (gSaveContext.save.time > CLOCK_TIME(23, 30) || gSaveContext.save.time <= CLOCK_TIME(6, 0)) {
+                            Message_StartTextbox(play, 0x27F7, &this->actor);
+                            this->textId = 0x27F7;
+                        } else if (gSaveContext.save.weekEventReg[14] & 0x40) {
+                            Message_StartTextbox(play, 0x27DE, &this->actor);
+                            this->textId = 0x27DE;
+                        } else if (func_80AE9B8C() == 2) {
+                            Message_StartTextbox(play, 0x27DF, &this->actor);
+                            this->textId = 0x27DF;
+                        } else if (func_80AE9B8C() == 1) {
+                            Message_StartTextbox(play, 0x27E0, &this->actor);
+                            this->textId = 0x27E0;
+                        } else {
+                            Message_StartTextbox(play, 0x27E1, &this->actor);
+                            this->textId = 0x27E1;
+                            break;
+                        }
+                        break;
```

Comment:
```suggestion
                        }
                        break;
```
Are both these `break;`'s needed?

Reply:
no